aana/alembic/versions/9cf76699210f_init.py

```diff
@@ -29,19 +70,20 @@ def upgrade() -> None:
             comment="Unique identifier for the media",
         ),
         sa.Column(
-            "media_type", sa.String(), nullable=True, comment="The type of media"
+            "media_type", sa.String(), nullable=False, comment="The type of media"
```

Comment:
Msst of the nullable attrs are changed to the false, It is okey to use base model (not extended version) to add item without the captions and other stuff?